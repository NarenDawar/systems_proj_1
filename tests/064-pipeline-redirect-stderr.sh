# some variations on redirecting to standard error + out
# should redirect only stdout 
writetostderr | rev
# should redirect both stdout and stderr
writetostderr |& rev
# should redirect both stdout and stderr
writetostderr >& .out5
cat .out5
rm .out5
