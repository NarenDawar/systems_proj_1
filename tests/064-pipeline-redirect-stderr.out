tuodts ot siht gnitirw
rredts ot siht gnitirw
tuodts ot siht gnitirw
writing this to stderr
writing this to stdout
